libs/nx/src/nx.ts

```diff
@@ -28,51 +28,71 @@ interface WorkspaceProject {
 }
 
 export async function getNxProjects(cwd: string): Promise<WorkspaceProject[]> {
+  const nxProjects = await getNxProjectJsonProjects(cwd);
+  const workspaceProjects = await getNxWorkspaceProjects(cwd);
+
+  const relevantWorkspaceProjects = workspaceProjects.filter(
+    (proj) => nxProjects.find((nested) => nested.name === proj.name) === null
```

Comment:
@EladBezalel did you want me to use `== null` here? I did put 3 equal signs out of habit 🤔 
```suggestion
    (proj) => nxProjects.find((nested) => nested.name === proj.name) == null
```

Reply:
Just a habit, 2 equal signs captures both null and undefined

Reply:
anyhow, should be either `== null` or `=== undefined`